### ISSUE TITLE
(WCAG 2.1) Change in Focus outline is barely visible on input fields

### ISSUE THREAD

**Issue by dkatzz** [civiform/civiform#5532]
OBSERVED RESULTS:

- The change from black to blue focus outline is hard to see, especially on comboboxes where there's no cursor like there is in input text boxes.
- Applicable for all form fields.
- ![Screenshot](https://github.com/civiform/civiform/assets/86739416/1ffe9cd9-ee1d-402e-8465-6be31d640a85)
- ![Screenshot](https://github.com/civiform/civiform/assets/86739416/cc9c1111-9ac9-4b21-b4cf-f1a0bc276507)

EXPECTED RESULTS:

- The focus indicator outline should meet a minimum contrast ratio of 3:1 to the background.
- Keyboard focus indicator should be visually distinct. In this case, implementing the keyboard focus indicator larger than the default black border can improve visibility.

STEPS TO REPRODUCE:

- Navigate to [URL](https://demo.civiform.dev/)
- Navigate to and activate the ‘Apply’ for the Comprehensive sample program.
- Navigate to and activate “What is your address?”.
- Navigate to each form field using keyboard TAB or SHIFT TAB key.
- Observe the focus outline for input fields and comboboxes.

ADDITIONAL INFORMATION:

- [ W3org Reference - 2.4.11: Focus Appearance (Minimum)](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance-minimum.html)

ENVIRONMENT DETAILS:

- Device(s) (brand, OS version, test key or release key): Windows 10 Enterprise (64-bit),MacOS Ventura Version 13.4
- Screen reader name and version: JAWS 2022.2211.7, NVDA 2022.3, VoiceOver
- Browser name and version: Chrome Version 113.0.5672.93

**Comment by melanie-exygy** [comment]
Once #5662 is completed, our focus color will be #2491ff, which is the USWDS default.  This will give us a contrast ratio of 3.19 in respect to the white background, which is acceptable.

![image](https://github.com/civiform/civiform/assets/134556723/eabed08a-f7e1-4623-9e8f-a2ce5a63fd42)

I propose closing this issue once #5662 is complete and adding a note to #5533 to ensure that the focus color is not changed to one that doesn't contrast enough with the white background.
